bugbug/utils.py

```diff
@@ -3,11 +3,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import os
+
 import numpy as np
+import taskcluster
```

Comment:
You'll need to add taskcluster to the requirements.txt now, since we need it to run bugbug